spec/controllers/vacancies_controller_spec.rb

```diff
@@ -60,6 +60,33 @@
           subject
         end
       end
+
+      context 'search audiotor' do
+        let(:params) do
+          {
+            subject: 'foo'
+          }
+        end
+
+        it 'should call the search auditor' do
+          expect(AuditSearchEventJob).to receive(:perform_later)
+
+          subject
+        end
+
+        it 'should not call the search auditor if its a smoke test' do
+          cookies[:smoke_test] = 1
```

Comment:
Forgive my knowledge of rails.  Does this get reset between tests?

Reply:
Yes it does, according to my tests